index.js

```diff
@@ -2,26 +2,32 @@ import './style.scss';
 import $ from "jquery";
 import validate from "jquery-validation/dist/jquery.validate.min.js";
 
-var d = new Date();
+const d = new Date();
+const postcode = document.querySelector("#postcode");
+postcode.addEventListener("keydown", function() {
+	const isNotNumber = isNaN(Number(postcode.value));
```

Comment:
Wondering if this makes more sense as a 4 digit number regex.

Reply:
Possibly? I played around with regexes but I couldn't figure out how to get it to check on the whole string, e.g. it was returning a 'match' for '4467fg' because it matched the first 4 characters. Do you have a pointer on how it might be done?

Reply:
Updated with trim() and regex exp 👍